handsontable/src/plugins/copyPaste/copyPaste.js

```diff
@@ -24,6 +24,7 @@ Hooks.getSingleton().register('afterCopy');
 
 export const PLUGIN_KEY = 'copyPaste';
 export const PLUGIN_PRIORITY = 80;
+const CONFIG_KEYS = ['fragmentSelection'];
```

Comment:
`copyPaste` is the `PLUGIN_KEY` which is always checked in the base plugin, `CONFIG_KEYS` are meant to be additional config keys (maybe I should change the name of that static to something like that?).

In this case, it would be more like "config keys, that can affect this plugin when updated" like `fragmentSelection` can affect `copyPaste` (it caches that value in `enablePlugin`).

Reply:
Ah, ok. I lead myself up the garden path then. To make this more obvious maybe we should rename that option. Such as `SETTING_KEYS_DEPS` or something similar with the `DEPS` keyword?  🤔